src/component/form/CaseElement.tsx

```diff
@@ -89,19 +105,65 @@ export const getExitName = (
     // Don't reassign func params
     let newExitName = exitName;
 
-    // Some operators don't expect args
-    if (newArgList.length >= 0 && !operatorConfig.categoryName) {
+    if (newArgList.length > 0 && !operatorConfig.categoryName) {
         newExitName = composeExitName(operatorConfig.type, newArgList);
-    } else {
+    } else if (operatorConfig.categoryName) {
+        // Some operators don't expect args
         // Use the operator's default category name
         ({ categoryName: newExitName } = operatorConfig);
     }
 
     return newExitName;
 };
 
-export const hasArgs = (args: string[] = []): boolean =>
-    args.length > 0 && args[0].trim().length > 0;
+/**
```

Comment:
helper to isolate `min` and `max` values from the arguments array of a case containing the `has_number_between` operator